tfctl/resume.go

```diff
@@ -28,6 +38,25 @@ func (c *CLI) Resume(out io.Writer, resource string) error {
 	return nil
 }
 
+func resumeAllReconciliation(ctx context.Context, kubeClient client.Client) error {
+	terraformList := &infrav1.TerraformList{}
+	if err := kubeClient.List(ctx, terraformList); err != nil {
```

Comment:
Does this list them all in all namespaces?

Reply:
https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/client#ListOptions